app.js

```diff
@@ -22,4 +27,25 @@ app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+   //res.render('beers');
+   punkAPI.getBeers().then(beersArray => {
+     res.render('beers', {arrayObject: beersArray })
+     console.log(beersArray)
+    }).catch(error => {
+      console.log(error)
+    });
+});
+
+app.get('/random-beer', (req, res) => {
+  punkAPI.getRandom().then(randomBeer => {
+    res.render('random-beer', {beerObject: randomBeer})
```

Comment:
randomBeer is an array of one object. So you could pass it to render like so randomBeer[0]. Also since it is an object by itself, you don't need to wrap it in an object.

res.render('random-beer', randomBeer[0])